### PR TITLE
Update ly.m3u

### PR DESCRIPTION
**Pull request by link8k**

### CODE REVIEW

File: channels/ly.m3u [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/mzwdt39.jpg" group-title="News",Libya 218 News
 http://95.85.47.43/libya218news/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/mzwdt39.jpg" group-title="News",Libya 218 News
+https://stream.218tv.net/libya218news/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://pbs.twimg.com/profile_images/953959542928412673/wYWx5MRY.jpg" group-title="",Libya Al Ahrar TV
+https://video.zidivo.com/live983/GrtjM_FNGC/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="http://www.elahmad.com/tv/images/libyaalhadath.jpg" group-title="",Libya Al Hadath
+https://master.starmena-cloud.com/hls/hd.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://www.banat5.com/wp-content/uploads/2019/10/channel-8.jpg" group-title="",Libya Almostaqbal
+http://livedroos.com:8080/live/Libyah-Mostaqbal/tracks-v1a1/mono.m3u8
```
Comment (themak1985) [inline channels/ly.m3u:17]: Not working.

**Review by themak1985 — COMMENTED** [review]
One stream isn't working, otherwise looks good @link8k

File: channels/ly.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/ly.m3u:17]: Maybe a non 24-7 stream. Maybe @link8k should try on its own and come back with proof (image or stream data)

Comment (link8k) [reply channels/ly.m3u]: Apologies, I remember testing them (maybe it went down). I will add https://streamtest.in/ links next time

**Pull request merged**